support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx

```diff
@@ -192,9 +184,9 @@ const StripeForm = (props: StripeFormPropTypes) => {
     });
   };
 
-  const setupRecurringHandlers = (): void => {
+  const setupRecurringHandlers = async () => {
```

Comment:
I'm interested to know whether we need to use `async` / `await` here? I see we're calling an async function `fetchPaymentIntent` but we're not waiting on its return result, so I think what will happen here is  `setupRecurringHandlers` will return `undefined` once `fetchPaymentIntent('dummy)` has resolved?

We're also not waiting on `setupRecurringHandlers` to resolve asynchronously when we call it [here](https://github.com/guardian/support-frontend/blob/a469c698620b3c1840b31cc5ddaea45b6825b664/support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx#L281).

Reply:
I think this was just something I did while trying to get the tests to play nicely with all the background async stuff the component does; it doesn't seem to be necessary though so I'll revert.